### ISSUE TITLE
siarplotdata() for one group

### ISSUE THREAD

**Issue by VirginiaMorera** [AndrewLJackson/siar#6]
Version: 4.2
I detected a possible bug in siarplotdata() for the single group version. Aparently takes the column of the grouping variable as the first isotope so siarplotdata(modeldemo, iso=c(1,2) produces 
![bugexample1](https://cloud.githubusercontent.com/assets/11058769/6246686/0724f0ca-b76c-11e4-9c94-09a246f3b7b2.png)
In my example N is the 1rst isotope, and siar is plotting it as the second, and "Code" as the first. As a consequence, the second isotope, C, gets plotted as the third like in siarplotdata(modeldemo, iso=c(2,3) 
![bugexample2](https://cloud.githubusercontent.com/assets/11058769/6246739/86aa93cc-b76c-11e4-8b77-9bd64ad0221b.png)
The sources are correctly plotted, and the axis values seem to correspond the the actual isotopic values of the sources, the problem is only in representing the consumers. 

This is also an issue with only two isotopes, but it disappears in the multigroup version.

**Comment by AndrewLJackson** [comment]
great example illustrating. i will take a look when i get a chance, but its going to be April at the earliest I'm afraid.

**Comment by VirginiaMorera** [comment]
Haven't had time to try it yet but this is probably fixed in version 4.2.2 as it fixes some bug regarding data from a single group of consumers?

**Comment by AndrewLJackson** [comment]
it might not be. i will try take a look tomorrow. Is this happening with the demo files too? https://github.com/AndrewLJackson/SIAR-examples-and-queries/tree/master/siar-mixing-models

**Comment by AndrewLJackson** [comment]
this works for me in the demo_script.r file using the geese1.txt consumer file in place of geese2.txt. Can you check it works for you and i will close this issue. thanks.

**Comment by xiaoweige2** [comment]
Have you solved the problem of the bug in one group? I have the same problem when i used stable isotope mixing model in R.

**Comment by VirginiaMorera** [comment]
Can't remember now how I fixed, or if I did. But in any case, SIAR has
stopped being maintained since there is another package with the same
functionality but better, updated modelling techniques, plotting functions
and so on. Check out https://github.com/andrewcparnell/simmr for more info,
code and vignettes.

Virginia Morera
PhD Student
Department of Animal Biology
University of Barcelona

Aquest correu electrònic i els annexos poden contenir informació
confidencial o protegida legalment i està adreçat exclusivament a la
persona o entitat destinatària. Si no sou  el destinatari final o la
persona encarregada de rebre’l, no esteu autoritzat a llegir-lo,
retenir-lo, modificar-lo, distribuir-lo, copiar-lo ni a revelar-ne el
contingut. Si heu rebut aquest correu electrònic per error, us preguem que
n’informeu al remitent i que elimineu del sistema el missatge i el material
annex que pugui contenir. Gràcies per la vostra col·laboració.

Este correo electrónico y sus anexos pueden contener información
confidencial o legalmente protegida y está exclusivamente dirigido a la
persona o entidad destinataria. Si usted no es el destinatario final o la
persona encargada de recibirlo, no está autorizado a leerlo, retenerlo,
modificarlo, distribuirlo, copiarlo ni a revelar su contenido. Si ha
recibido este mensaje electrónico por error, le rogamos que informe al
remitente y elimine del sistema el mensaje y el material anexo que pueda
contener. Gracias por su colaboración.

This email message and any documents attached to it may contain
confidential or legally protected material and are intended solely for the
use of the individual or organization to whom they are addressed. We remind
you that if you are not the intended recipient of this email message or the
person responsible for processing it, then you are not authorized to read,
save, modify, send, copy or disclose any of its contents. If you have
received this email message by mistake, we kindly ask you to inform the
sender of this and to eliminate both the message and any attachments it
carries from your account.Thank you for your collaboration.

2017-10-15 15:10 GMT+02:00 xiaoweige2 <notifications@github.com>:
